### PR TITLE
Admin access for course staff fall 17

### PR DESCRIPTION
**Pull request by vinitra**
added admin permissions in datahub config for data 8 gsis, professors, and infrastructure team

### CODE REVIEW

File: datahub/config.yaml [hunk 1]
```diff
@@ -23,3 +23,45 @@ singleuser:
     extraVolumeMounts:
       - name: home
         mountPath: /home/jovyan
+
+admin:
+  users:
+    # infrastructure
```
Comment (yuvipanda) [inline datahub/config.yaml]: can you align the comments with the user name list?

Comment (vinitra) [inline datahub/config.yaml]: Done! Added an additional commit.

File: datahub/config.yaml [hunk 2]
```diff
@@ -23,3 +23,45 @@ singleuser:
     extraVolumeMounts:
       - name: home
         mountPath: /home/jovyan
+
+admin:
+  users:
+  # infrastructure
```
Comment (yuvipanda) [inline datahub/config.yaml]: Can you align the '#' with the '-' in the list? :D That's what I meant I think. Sorry!

Comment (vinitra) [inline datahub/config.yaml]: Done!

**Comment by yuvipanda** [comment]
Cool! I merge this...

**Comment by yuvipanda** [comment]
You can see the deploying happening at https://travis-ci.org/berkeley-dsep-infra/datahub/builds/265344835

**Comment by yuvipanda** [comment]
And it's done!

**Comment by yuvipanda** [comment]
@vinitra @gunjanbaid so if you go to datahub-staging.yuvi.in I see we are all admins now. Can you verify, and then I can show how to deploy this to prod?

**Pull request merged**